### DISCUSSION TITLE
Problem extracting urls and image urls using the FetchNode

### DISCUSSION THREAD

**Discussion by DenisMarasescu** [ScrapeGraphAI/Scrapegraph-ai#452] [Q&A]
In the docusaurus mentioned in the readme (specifically https://scrapegraph-doc.onrender.com/docs/Nodes/fetch_html_node), it is shown that the fetch_node should return the fetched_content, the link_urls and the image_urls. 

Unfortunately, they are not. I wonder why and, especially, how can I retrieve the urls and the image urls without this function.

**Comment by VinciGit00** [comment]
please look at the state and use a debugger
